web/yo/app/scripts/directives/reviewpanel.js

```diff
@@ -312,9 +312,7 @@ angular.module('oncokbApp')
                                 if(lastReviewedLevel !== null) {
                                     $scope.tt.level.setText(lastReviewedLevel);
                                 }
-                                if(lastReviewedPropagation !== null) {
```

Comment:
we shouldn't do the check. If lastReviewedPropagation is null, we should set it back as null.